build_tools/azure/test_script.sh

```diff
@@ -42,19 +43,19 @@ show_installed_libraries
 TEST_CMD="python -m pytest --showlocals --durations=20 --junitxml=$JUNITXML -o junit_family=legacy"
 
 if [[ "$COVERAGE" == "true" ]]; then
-    # Note: --cov-report= is used to disable to long text output report in the
+    # Note: --cov-report= is used to disable too long text output report in the
     # CI logs. The coverage data is consolidated by codecov to get an online
     # web report across all the platforms so there is no need for this text
     # report that otherwise hides the test failures and forces long scrolls in
     # the CI logs.
-    export COVERAGE_PROCESS_START="$BUILD_SOURCESDIRECTORY/.coveragerc"
+    export COVERAGE_PROCESS_START="$CHECKOUT_FOLDER/.coveragerc"
```

Comment:
Similar comment as https://github.com/scikit-learn/scikit-learn/pull/31941/files#r2276870682. It feels slightly preferable to rely on the assumption that the current working directory is the scikit-learn repo root folder.